targets/TARGET_Samsung/TARGET_SIDK_S5JS100/spi_api.c

```diff
@@ -37,7 +38,7 @@ typedef struct {
 
 int s5js100_configgpio(uint32_t cfgset);
 
-static const PinMap PinMap_SPI_SCLK[] = {
+/*static const PinMap PinMap_SPI_SCLK[] = {
```

Comment:
why are these being removed?

It should be completely removed not just comment out

Reply:
Yes, we will to remove it.